tdp/core/service_manager.py

```diff
@@ -101,12 +110,13 @@ def initialize_variables(self):
                     pass
 
     @staticmethod
-    def initialize_service_managers(dag, services_directory):
+    def initialize_service_managers(dag, services_directory, override_folder=None):
         """get a dict of service managers, initialize all services if needed
 
         Args:
             dag (Dag): operations' DAG
             services_directory (Union[str, Path]): path of the tdp vars
+            override_folder (Optional(str, Path)): path of tdp vars overrides
```

Comment:
`override_folder` is a `str` or a `Path` so `Optional[str | Path]` no ?

Reply:
Corrected documentation